e2e/print-new-node/rector.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+use Rector\Config\RectorConfig;
+use Rector\Php80\Rector\ClassMethod\AddParamBasedOnParentClassMethodRector;
+
+return static function (RectorConfig $rectorConfig): void {
+    $rectorConfig->paths([
+        __DIR__ . '/src/TestClass.php',
+        __DIR__ . '/src/ExtendingTestClass.php',
+    ]);
+
+    $rectorConfig->rule(AddParamBasedOnParentClassMethodRector::class);
```

Comment:
Could you add this as a direct test to `AddParamBasedOnParentClassMethodRector`? One file is a test fixture, other is a `/Source`. It would be easier to fix directly.

Reply:
it only reproducible via e2e unfortunatelly...